programs/ziti-edge-tunnel/package/systemd/ziti-edge-tunnel.sh.in

```diff
@@ -8,25 +8,25 @@ set -eu
 
 for JWT in @ZITI_IDENTITY_DIR@/*.jwt; do
     # handle case where JWT='@ZITI_IDENTITY_DIR@/*.jwt'
-    [ -e ${JWT} ] || {
+    [ -e "${JWT}" ] || {
         echo "NOTICE: no new JWT files in ${JWT}" >&2
         continue
     }
-    [ -r ${JWT} ] || {
+    [ -r "${JWT}" ] || {
         echo "WARN: skipping unreadable JWT file in ${JWT}" >&2
         continue
     }
-    [ -s ${JWT} ] || {
+    [ -s "${JWT}" ] || {
         echo "WARN: skipping empty JWT file in ${JWT}" >&2
         continue
     }
     # equivalent to BASH's ${JWT%.jwt}.json
-    CONFIG="$(echo ${JWT}|sed -E 's|(.*).jwt|\1.json|')"
+    CONFIG="$(echo "${JWT}" | sed -E 's|(.*).jwt|\1.json|')"
     if @CPACK_BIN_DIR@/@SYSTEMD_SERVICE_NAME@ enroll --jwt ${JWT} --identity ${CONFIG}; then
-        mv --force ${JWT} ${JWT}.enrolled
-        echo "INFO: enrolled $(basename ${JWT}) in ${CONFIG}"
+        mv --force "${JWT}" "${JWT}.bak"
```

Comment:
@qrkourier See: https://github.com/openziti/ziti-tunnel-sdk-c/blob/cddb16d115b61047d6e788b285110836d583e0fb/programs/ziti-edge-tunnel/ziti-edge-tunnel.c#L1063

Reply:
OK, so it'll ignore files with a name suffix like .bak or .original. I'd already pushed a change to remove the consumed tokens but I think it got orphaned in the 0.17.36 trunk.

Reply:
I'd be happy to enhance this PR to include your implementation if you can share it.